src/docs/sdk/check-ins.mdx

```diff
@@ -24,6 +24,11 @@ The following fields exist:
 
 : **String, required**. Check-In ID (unique and client generated).
 
+  This may be provided as a empty UUID (128 bit zero value) to indicate to
```

Comment:
Do you expect the SDKs to implement this feature?

Reply:
At the moment no I would expect almost all SDKs to maintain their checkin IDs for each run. This is to support the simplified CURL style API as outlined in https://github.com/getsentry/sentry/issues/49664

Reply:
The linked PR mentions `null`, do you suggest `00000...`?

Reply:
I think @mitsuhiko's description is slightly misleading, it is not nullable, but it passes a empty value instead. You can see it in the tests here https://github.com/getsentry/relay/pull/2153/files#diff-2b7b3259fa82d6066fa1268aed58cf7a8bac4e9317e275b7601cc916b1201e4bR56